src/ngraph/frontend/onnx_import/op/conv_integer.cpp

```diff
@@ -0,0 +1,87 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "op/conv_integer.hpp"
+#include "ngraph/builder/make_constant.hpp"
+#include "ngraph/builder/quantization/quantized_linear_convolution.hpp"
+#include "ngraph/frontend/onnx_import/exceptions.hpp"
+#include "ngraph/frontend/onnx_import/utils/convpool.hpp"
+
+using namespace ngraph::builder;
+
+namespace ngraph
+{
+    namespace onnx_import
+    {
+        namespace op
+        {
+            namespace set_1
+            {
+                NodeVector conv_integer(const Node& node)
+                {
+                    const NodeVector& inputs = node.get_ng_inputs();
+                    auto num_inputs = inputs.size();
+                    auto input = inputs.at(0);
+                    auto filters = inputs.at(1);
+
+                    int64_t groups{node.get_attribute_value<int64_t>("group", 1)};
+                    ASSERT_VALID_ARGUMENT(node, (groups == 1))
```

Comment:
What refrain us from doing sth similar to this: https://github.com/NervanaSystems/ngraph/blob/master/src/ngraph/frontend/onnx_import/op/conv.cpp#L51-L93 ?

Reply:
Nothing, I just don't think `groups` are actually used in real-world models any more.